Makefile

```diff
@@ -199,7 +199,7 @@ check_bundledir:
 	@$(call check_defined, BUNDLE_DIR, "Embedding bundle requires BUNDLE_DIR set to a directory containing CRC bundles for all hypervisors")
 
 embed_bundle: LDFLAGS += $(BUNDLE_EMBEDDED)
-embed_bundle: clean cross $(GOPATH)/bin/binappend-cli check_bundledir $(HYPERKIT_BUNDLENAME) $(HYPERV_BUNDLENAME) $(LIBVIRT_BUNDLENAME)
-	$(GOPATH)/bin/binappend-cli write $(BUILD_DIR)/linux-amd64/crc $(LIBVIRT_BUNDLENAME)
-	$(GOPATH)/bin/binappend-cli write $(BUILD_DIR)/macos-amd64/crc $(HYPERKIT_BUNDLENAME)
-	$(GOPATH)/bin/binappend-cli write $(BUILD_DIR)/windows-amd64/crc.exe $(HYPERV_BUNDLENAME)
+embed_bundle: clean cross build-aux/crc-embedder check_bundledir $(HYPERKIT_BUNDLENAME) $(HYPERV_BUNDLENAME) $(LIBVIRT_BUNDLENAME)
+	build-aux/crc-embedder embed --goos=darwin --bundle-dir=$(BUNDLE_DIR) $(BUILD_DIR)/macos-amd64/crc
```

Comment:
wondering why this is `build-aux`. is this intended as a place for more build related tooling?

Reply:
If we need more build tools in the future, yes they could go there. But this is just a name I picked from my autotools day, I don't mind changing it to something different.

Reply:
just wondering... not a problem. it almost suggest you have more tools on mind,... if so, what are those?

Reply:
nope, definitely no additional tool in mind :) Not even sure it would be convenient to build 2 binaries from the same dir with go.

Reply:
actually, the patter for that is `cmd/build-aux`, just like `cmd/crc`. the second `cmd` in `cmd/crc/cmd` is actually for cobra/viper.